when turning examples ON, then I get build failures
see below
```
-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api1x_core/legacy-decode/CMakeLists.txt:65 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build legacy-decode without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api1x_core/legacy-vpp/CMakeLists.txt:65 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build legacy-vpp without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api1x_core/legacy-encode/CMakeLists.txt:65 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build legacy-encode without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-decode/CMakeLists.txt:69 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-decode without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-sharing-vaapi/CMakeLists.txt:72 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-sharing-vaapi-export
  without libva support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-sharing-vaapi/CMakeLists.txt:72 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-sharing-vaapi-import
  without libva support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-decvpp/CMakeLists.txt:69 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-decvpp without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-encode/CMakeLists.txt:68 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-encode without libva
  support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-encode-jpeg/CMakeLists.txt:68 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-encode-jpeg without
  libva support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-transcode/CMakeLists.txt:69 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-transcode without
  libva support


-- Checking for module 'libva>=1.2'
--   Package 'libva', required by 'virtual:world', not found
-- Checking for module 'libva-drm>=1.2'
--   Package 'libva-drm', required by 'virtual:world', not found
CMake Error at examples/api2x/hello-vpp/CMakeLists.txt:78 (message):
  libva not found: set LIBVA_SUPPORT=OFF to build hello-vpp without libva
  support
  ```
so at least libva needs to be on that list

--- a/debian/control
+++ b/debian/control
@@ -4,6 +4,7 @@ Priority: optional
 Maintainer: Timo Aaltonen <tjaalton@debian.org>
 Build-Depends: debhelper-compat (= 13),
  cmake,
+ libva-dev,
  pkgconf,
 Standards-Version: 4.7.0
 Homepage: https://github.com/intel/libvpl

--- a/debian/libvpl2.install
+++ b/debian/libvpl2.install
@@ -1,2 +1,1 @@
 usr/lib/*/libvpl.so.*
-usr/share/vpl/licensing

--- a/debian/upstream/metadata
+++ b/debian/upstream/metadata
@@ -1,0 +1,6 @@
+Bug-Database: https://github.com/intel/libvpl/issues
+Bug-Submit: https://github.com/intel/libvpl/issues/new
+Changelog: https://github.com/intel/libvpl/blob/master/CHANGELOG.md
+#Documentation: https://github.com/intel/libvpl/wiki
+Repository-Browse: https://github.com/intel/libvpl
+Repository: https://github.com/intel/libvpl.git

